src/utils/Users.js

```diff
@@ -0,0 +1,133 @@
+const niceJoin = array =>
```

Comment:
It would be nice to use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ListFormat once it becomes supported in Firefox :) Works in Chrome for now.

Reply:
Yes, that's exactly what I want :sob: